### PR TITLE
Responsive website recoded 

### PR DESCRIPTION
**Pull request by Jfalzarano888**
Here is the remastered code so the website contracts and expands depending on the browser window. No break points have been created yet, but those should be added in the next few updates.

### CODE REVIEW

File: HTML/contact.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                  <img src="svgs/partyphone.svg" alt="computer"/>
+                  <div class="center-header">
+                      <h1> Phone: 908-210-2566 </h1>
+                  </div>
+                  <div class="line"></div>
+                </section>
+        </div>
+        <div class="thanks-box-center">
+          <div class="logo-wrap">
+              <img src="svgs/logo.svg" alt="computer"/>
```
Comment (ttaghiyev) [inline HTML/contact.html]: for relative paths, always start then with `./` so `svgs/logo.svg` becomes `./svgs/logo.svg`

File: HTML/index.html [hunk 2]
```diff
@@ -6,37 +6,109 @@
...
+          </nav>
 
+          <section class="top-main">
+              <div class="feature-row">
 
+                  <div class="feature feature1">
+                      <div class="headline">
+                          <h2> Quality Design </h2>
+                      </div>
+                      <img src="svgs/monitor.svg" alt="computer" margin: "0 auto"/>
```
Comment (ttaghiyev) [inline HTML/index.html]: `margin: 0 auto` - cant use styles like this, this has no meaning, I'd take it out (that goes for all of em)

File: HTML/styles/main.scss [hunk 3]
```diff
@@ -1,80 +1,325 @@
...
-.feature3 {
-  display: $inline;
-  height: 442px;
-  width: (1000/3) * 1px;
-  background-color: green;
+
+.phone-image {
+  width: 100%;
+  height: 100%;
+  background: url('../svgs/phone.svg') no-repeat center;
```
Comment (ttaghiyev) [inline HTML/styles/main.scss]: you probably want that second 'center' in there too

File: HTML/contact.html (hunk 1 above)
Comment (Jfalzarano888) [reply HTML/contact.html]: I'm gonna jump that gorde.... you... BITCH --

**Comment by ttaghiyev** [comment]
Looks good, sort out the stuff I called out, and I would do the following two things next:

1. remove the nonsense files from git (via .gitignore) like .DS_Store and caches
2. take out the hat2.svg from the top level directory. you have it in images so you shouldnt need it there.

otherwise looks good

**Pull request merged**